src/main/webapp/styles/index.css

```diff
@@ -23,6 +23,15 @@ body {
   font-weight: bolder;
 }
 
+#title {
+  text-align: center;
+  padding: 20px;
```

Comment:
[Nit] Replace px with some relative unit (vw, vh, vmin, vmax) to adapt to different screen sizes.